### DISCUSSION TITLE
Razer setup help

### DISCUSSION THREAD

**Discussion by jhollis505** [hyperion-project/hyperion.ng#1460] [Q&A]
I am using a razer chroma argb controller with 4 strips of ws2812b light strips connected to said razer controller. I need help getting the web config to recognize the razer controller.

**Comment by Lord-Grey** [comment]
I tried to find how to address the razer rgb Controller via Razer‘s REST API, but I did not find anything.
I currently suppose that it is not possible and therefore you cannot use this device with Hyperion.

**Comment by jhollis505** [comment]
Is there any controller like this that I could use?
